tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSSharedDurableConsumerTest.java

```diff
@@ -138,4 +140,46 @@ public void testSharedDurableConsumerWithArtemisClientAndAMQPClient() throws Exc
       testSharedDurableConsumer(connection, connection2);
 
    }
+
+   @Test(timeout = 30000)
+   public void testSharedDurableConsumerWithSelector() throws JMSException {
```

Comment:
```suggestion
   public void testSharedDurableConsumerWithSelectorChange() throws JMSException {
```
Clarify it doesnt test the selector behaviour itself, only the effect of changing it.